ci/dev/watch.ts

```diff
@@ -173,30 +166,27 @@ class Watcher {
           console.log("[plugin]", original)
         }
         if (line.includes("Watching for file changes")) {
-          bundle.then(restartServer)
+          bundleBrowserCode(browserFiles)
+          restartServer()
         }
       })
     }
   }
+}
 
-  private createBundler(out = "dist"): FixMeLater {
-    return new (Parcel as FixMeLater)({
-      entries: [
-        path.join(this.rootPath, "src/browser/register.ts"),
-        path.join(this.rootPath, "src/browser/serviceWorker.ts"),
-        path.join(this.rootPath, "src/browser/pages/login.ts"),
-        path.join(this.rootPath, "src/browser/pages/vscode.ts"),
-      ],
-      cacheDir: path.join(this.rootPath, ".cache"),
-      logLevel: 1,
-      defaultConfig: require.resolve("@parcel/config-default"),
-      defaultTargetOptions: {
-        publicUrl: ".",
-        shouldOptimize: !!process.env.MINIFY,
-        distDir: path.join(this.rootPath, out),
-      },
-    })
-  }
+function bundleBrowserCode(inputFiles: string[]) {
+  console.log(`[browser] bundling...`)
+  inputFiles.forEach(async (path: string) => {
+    const outputPath = path.replace(".js", ".browserified.js")
+    browserify()
+      .add(path)
+      .bundle()
+      .on("error", function (error: Error) {
+        console.error(error.toString())
+      })
+      .pipe(fs.createWriteStream(outputPath))
```

Comment:
Should we add a log `console.log(`[browser] bundled ${path} to `${outputPath}`)?

Reply:
IMO the "bundling..." and "done bundling" logs are sufficient!